pyEpiabm/pyEpiabm/__init__.py

```diff
@@ -1,44 +1,18 @@
-
 #
 # Root of the pyEpiabm module.
 # Provides access to all shared functionality (classes, simulation, etc.).
 #
-"""pyepiabm is the python backend for epiabm - an Epidemiological
+
+"""pyEpiabm is the python backend for epiabm - an Epidemiological
 Agent Based Modelling software package.
 It contains functionality for setting up a population, and tracking the
 evolution of a virus across it, with various visualisation methods.
 """
 
-# Core
-from .person import Person
-from .population import Population
-from .cell import Cell
-from .microcell import Microcell
-from .household import Household
-from .place import Place
-from .parameters import Parameters
-
-# Properties
-from .infection_status import InfectionStatus
-from .place_type import PlaceType
-
-# Routines
-from .covidsim_helpers import CovidsimHelpers
-from .toy_population_config import ToyPopulationFactory
-
-# Sweeps
-from .abstract_sweep import AbstractSweep
-from .host_progression_sweep import HostProgressionSweep
-from .household_sweep import HouseholdSweep
-from .initial_infected_sweep import InitialInfectedSweep
-from .place_sweep import PlaceSweep
-from .queue_sweep import QueueSweep
-from .update_place_sweep import UpdatePlaceSweep
-
-# Example Simulations
-from .simulation import Simulation
+# from pyEpiabm.property.infection_status import InfectionStatus
 
-# Data collection
-from ._compartment_counter import _CompartmentCounter
-from ._csv_dict_writer import _CsvDictWriter
-from ._csv_writer import _CsvWriter
+from . import core
```

Comment:
```suggestion
from . import core
from .core.cell import Cell
```
etc.
Did you consider making the core classes available right under pyepiabm for ease of access?
It's fine as it is, but I would also be happy if core was just a category for our internal use and its classes were available directly.

Reply:
Hmm, how far would we extend this? I thought one of the main motivations in reorganising the init files specifically was to avoid cluttering the pyEpiabm namespace (so we wouldn't want everything in there) and I think it could become hard to decide what is important enough to be in the pyEpiabm namespace (I know this is in fact an issue larger modules like scipy come across).

Personally I think I tend towards leaving it as is, rather than trying to make an arbitrary decision on which modules go in the top level init and introducing inconsistency on import methods, however very happy to chat about this @rcw5890 ?

Reply:
> Hmm, how far would we extend this? I thought one of the main motivations in reorganising the init files specifically was to avoid cluttering the pyEpiabm namespace (so we wouldn't want everything in there) and I think it could become hard to decide what is important enough to be in the pyEpiabm namespace (I know this is in fact an issue larger modules like scipy come across).
> 
> Personally I think I tend towards leaving it as is, rather than trying to make an arbitrary decision on which modules go in the top level init and introducing inconsistency on import methods, however very happy to chat about this @rcw5890 ?

It's not at all unprecedented to have a `core` subpackage but expose all its contents in the main namespace---as far as I know this is exactly what numpy, pandas, and sympy are all doing. I agree we will be repeatedly asking whether something is central enough to go there, but that's not insurmountable.
scipy is more an exception owing to its extremely broad subject coverage. They don't seem to put anything at all in the main namespace except scipy.test(), but all their subpackages are meaningful subcategories covering a specific topic much narrower in scope than scipy itself.

Though I don't disagree that not touching the main namespace has the appeal of consistency, so I'm happy either way---PR approved and I'd gladly hear any more ideas on this topic.

Reply:
Changed as requested - you can now import Cell from pyEpiabm directly (although not when within the pyEpiabm code itself as that would result in circular imports)